example/apps/test_security/tasks.py

```diff
@@ -7,13 +7,13 @@
     base=LoggedTask,
     bind=True,
     name='sum_task')
-def sum_task(self, task_id, a, b):
+def sum_task(self, a, b):
```

Comment:
task_id už není nutné přenášet, dá se získat z self.request.id, to je objekt, který drží kontext k danému tasku